File: desktop/GuestHousetry3/src/entities/Claim.java

package entities;

import java.sql.Date;

public class Claim {
    private int id;
    private Date date;
    private String text;
    private String status ;
    private boolean isTreated;

    public Claim(int id, Date date, String text, String status, boolean isTreated) {
        this.id = id;
        this.date = date;
        this.text = text;
        this.status = status;
        this.isTreated = isTreated;
    }
    
    public Claim() {
        
    }

    public Claim(int id, Date date, String text, String status) {
        this.id = id;
        this.date = date;
        this.text = text;
        this.status = status;
    }

    public Claim(Date date, String text) {
        this.date = date;
        this.text = text;
    }

    public Claim(int id, Date date, String text) {
        this.id = id;
        this.date = date;
        this.text = text;
    }
    
    public int getId() {
        return id;
    }

    public Date getDate() {
        return date;
    }

    public String getText() {
        return text;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }

    public void setDate(Date date) {
        this.date = date;
    }

    public void setText(String text) {
        this.text = text;
    }

    @Override
    public int hashCode() {
        int hash = 7;
        hash = 41 * hash + this.id;
        return hash;
    }

    @Override
    public boolean equals(Object obj) {
        if (this == obj) {
            return true;
        }
        if (obj == null) {
            return false;
        }
        if (getClass() != obj.getClass()) {
            return false;
        }
        final Claim other = (Claim) obj;
        if (this.id != other.id) {
            return false;
        }
        return true;
    }

    @Override
    public String toString() {
        return "Claim{" + "id=" + id + ", date=" + date + ", text=" + text + ", status=" + status + '}';
    }

    public boolean isIsTreated() {
        return isTreated;
    }

    public void setIsTreated(boolean isTreated) {
        this.isTreated = isTreated;
    }

    
    
}

File: web/GuestHouseTry5/src/AppBundle/Entity/Claim.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Claim
 *
 * @ORM\Table(name="claim", indexes={@ORM\Index(name="fk_stayid", columns={"fk_claim_houseid"}), @ORM\Index(name="fk_claimerid", columns={"fk_claim_ownerrid"})})
 * @ORM\Entity
 */
class Claim
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $id;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="date", nullable=false)
     */
    private $date;

    /**
     * @var string
     *
     * @ORM\Column(name="text", type="text", length=65535, nullable=false)
     */
    private $text;

    /**
     * @var string
     *
     * @ORM\Column(name="status", type="text", length=65535, nullable=true)
     */
    private $status;

    /**
     * @var boolean
     *
     * @ORM\Column(name="istreated", type="boolean", nullable=false)
     */
    private $istreated = '0';

    /**
     * @var \House
     *
     * @ORM\ManyToOne(targetEntity="House")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="fk_claim_houseid", referencedColumnName="id")
     * })
     */
    private $fkClaimHouseid;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="fk_claim_ownerrid", referencedColumnName="id")
     * })
     */
    private $fkClaimOwnerrid;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param \DateTime $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }

    /**
     * @return string
     */
    public function getText()
    {
        return $this->text;
    }

    /**
     * @param string $text
     */
    public function setText($text)
    {
        $this->text = $text;
    }

    /**
     * @return string
     */
    public function getStatus()
    {
        return $this->status;
    }

    /**
     * @param string $status
     */
    public function setStatus($status)
    {
        $this->status = $status;
    }

    /**
     * @return boolean
     */
    public function isIstreated()
    {
        return $this->istreated;
    }

    /**
     * @param boolean $istreated
     */
    public function setIstreated($istreated)
    {
        $this->istreated = $istreated;
    }

    /**
     * @return \House
     */
    public function getFkClaimHouseid()
    {
        return $this->fkClaimHouseid;
    }

    /**
     * @param \House $fkClaimHouseid
     */
    public function setFkClaimHouseid($fkClaimHouseid)
    {
        $this->fkClaimHouseid = $fkClaimHouseid;
    }

    /**
     * @return \User
     */
    public function getFkClaimOwnerrid()
    {
        return $this->fkClaimOwnerrid;
    }

    /**
     * @param \User $fkClaimOwnerrid
     */
    public function setFkClaimOwnerrid($fkClaimOwnerrid)
    {
        $this->fkClaimOwnerrid = $fkClaimOwnerrid;
    }




}

